examples/raffle/index.rsh

```diff
@@ -122,8 +122,9 @@ export const main =
         const sponsort = declassify(_sponsort); });
       Sponsor.publish(sponsort);
       require(sponsortc == digest(sponsort));
-
-      const winningNo = (hwinner + (sponsort % howManyReturned)) % howManyReturned;
+      // CFX does like doing division with UInts );
+      // shared_impl.ts ln 356, Players bigNumber division is throwing it off
+      const winningNo = hwinner + sponsort;
```

Comment:
The change that you made is not correct. What you've changed it to will overflow

Reply:
The assertion of the original mod division remainder (sponsors % howManyReturned) % howmanyReturn 
 fails and points to a BigNumbers mod error, I kept thinking it was a issue w gas and storage but its how CFX handles bigNumber values im pretty sure

Reply:
That is very valuable to learn. We can't change the program though. Potentially we need to update how the Reach compiler outputs OR we need to get a problem report to Conflux. IDEA: Could you write a little Solidity program by hand that has the same problem when run on Conflux, but not on Eth?